packages/runtime-core/src/components/Teleport.ts

```diff
@@ -402,7 +402,7 @@ function updateCssVars(vnode: VNode) {
   const ctx = vnode.ctx
   if (ctx && ctx.ut) {
     let node = (vnode.children as VNode[])[0].el!
-    while (node !== vnode.targetAnchor) {
+    while (node && node !== vnode.targetAnchor) {
```

Comment:
This modification should have nothing to do with pr, I think this modification should be avoided